### DISCUSSION TITLE
thumb - floating or always on the key?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [manna-harbour/miryoku#261] [Q&A] [answered]
My index finger are almost always on T & N, but my thumbs are always floating unless i have to press any key. Is this how it supposed to be?

**Comment by extua** [comment]
I tend to keep my thumb above the middle thumb key for space/backspace when typing, and yeah like you I don't always have my thumb resting on the keys.

I have a flat/low profile board though, maybe it makes a difference with a more sculpted board which allows your thumb to rest in a pincer position?
What kind of board do you use?

**Reply by KAGEYAM4** [reply]
I am saving money for split keyboard, currently i am using my laptop keyboard using `kmonad`.

**Reply by extua** [reply]
When using my laptop keyboard I keep my hands hovering above the board and stab down with my fingers. I realise that's not the best typing posture though, you're _supposed_ to keep your fingers on the keys and only press down enough to activate the switch - ie. no bottoming out.

Personally I find the inner thumb key a bit difficult to hit, curling my thumb inwards doesn't feel natural.
Let us know how you get on with a split board!

**Comment by KAGEYAM4** [comment]
> Personally I find the inner thumb key a bit difficult to hit, curling my thumb inwards doesn't feel natural. Let us know how you get on with a split board!

Yes, same here, so what i did was edit the config and move the thumb key little bit outside. 

Here's a photo ->
![output](https://github.com/manna-harbour/miryoku/assets/75798544/5b90800e-a789-4cf9-b163-0f4389f11e82)


Left Home row - `A,R,S,T`
Right Home row - `N,E,I,O`

Left Thumb Home row - `2,3,4`
Right Thumb Home row - `9, -, 0`

**Comment by manna-harbour** [comment, marked as answer]
Thumbs should be resting over the home thumb keys, as the layout is designed to prioritise the primary thumb key, and to require only a 1u movement to reach the secondary and tertiary thumb keys.

Whether or not the thumbs are touching the key or hovering probably isn't so important, and will depend on hand / keyboard angles, etc., although I suspect it'd be better to have them in contact if it's comfortable.

If you have trouble finding the home thumb keys, adding homing bars or bumps etc. and keeping in contact when resting can help.
